stable/mediawiki/templates/tls-secrets.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.ingress.enabled }}
+{{- range .Values.ingress.secrets }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .name }}
```

Comment:
is this name ok?

Reply:
I think so

Reply:
yes, it is fine. I didn't realize it was inside a `range`

Reply:
👍